web/src/lib/components/asset-viewer/video-viewer/buttons/airplay-button.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+  import Icon from '$lib/components/elements/icon.svelte';
+  import { mdiCastVariant } from '@mdi/js';
+  import type { TooltipPlacement } from 'vidstack';
+  import Tooltip from '../tooltip.svelte';
+  type Props = {
```

Comment:
```suggestion

  type Props = {
```
Extremely minor nit 😅